solara/lab/hooks/dataframe.py

```diff
@@ -1 +1,2 @@
 from ..utils.dataframe import df_columns as use_df_column_names  # noqa: F401
+from ..utils.dataframe import df_rows as use_df_row_names  
```

Comment:
```suggestion
from ..utils.dataframe import df_rows
```
No need to alias it to use_df_row_names